src/compiler/crystal/macros/methods.cr

```diff
@@ -1077,6 +1082,22 @@ module Crystal
 
           value
         end
+      when "has_key?"
+        interpret_check_args do |key|
+          case key
+          when SymbolLiteral
+            key = key.value
+          when MacroId
+            key = key.value
+          when StringLiteral
+            key = key.value
+          else
+            raise "argument to has_key? must be a symbol or string, not #{key.class_desc}:\n\n#{key}"
```

Comment:
Please, format the exception message similarly to the other instances.

Reply:
i copied it from here: https://github.com/crystal-lang/crystal/pull/14890/files#diff-5d087db61d41eddf80efd1bb7dae3dc22c81b6ebe9e98cc36606480519248b6bR1057. I didn't notice that some other exceptions are using the `klass#method` notation.

BTW it seems that out of 39[^1] exceptions 6[^2] aren't referencing the class. Is this something that should be addressed in a followup PR? 


[^1]: `git grep "raise.*argument to " | wc -l`
[^2]: 39 - `git grep "raise.*argument to " | grep "#" | wc -l` = 6

Reply:
i reformatted the exception message: 221aaedf316b7c5248886c5342ead6323959590d